js/src/forum/components/DiscussionList.js

```diff
@@ -166,12 +217,20 @@ export default class DiscussionList extends Component {
   /**
    * Load the next page of discussion results.
    *
+   * @param isNext the page to load is the next page, false for previous page
    * @public
    */
-  loadMore() {
-    this.loading = true;
+  load(isNext = true) {
+    if (isNext) {
+      this.loading = true;
+      this.page = ++this.lastLoadedPage;
+    } else if (this.firstLoadedPage !== 1) {
+      this.loadingPrev = true;
+      this.page = --this.firstLoadedPage;
+      this.addResultsToBeginning = true;
```

Comment:
Doesn't this mean that weird things will happen when you first

I guess I am arguing not to have this value stored as state on the component, but rather as values closed over by the event handlers below.

Reply:
I don't remember what that code does, but I guess it could cause some issues.